src/QandA/QandA.jsx

```diff
@@ -13,15 +13,15 @@ class QandA extends React.Component {
     super(props);
     this.state = {
       questions: [],
-      product_id: ''
+      product_id: '19093'
     }
     this.fetchQuestions = this.fetchQuestions.bind(this);
     this.postQuestion = this.postQuestion.bind(this);
   }
 
 
   fetchQuestions() {
-    axios.get('qa/questions/?product_id=19093&page=1&count=30')
+    axios.get(`qa/questions/?product_id=${this.state.product_id}&page=1&count=30`)
```

Comment:
Great on setting that current id into state for better usability moving forward as product id's begin to shift.